api/api-doc/raft.json

```diff
@@ -62,6 +62,38 @@
                ]
             }
          ]
+      },
+      {
+         "path": "/raft/read_barrier",
+         "operations": [
+            {
+               "method": "POST",
+               "summary": "Triggers read barrier for the given Raft group",
```

Comment:
Maybe add some words on when this is useful?

Reply:
Updated the summary